marketpulse/main/models.py

```diff
@@ -41,32 +42,29 @@ def __unicode__(self):
         return '{0}, {1}'.format(self.shop_name, self.country)
 
 
-class Plan(models.Model):
-    """Mobile phone plan information."""
-
-    has_plan = models.BooleanField(default=False)
-    duration = models.IntegerField(default=None)
-    description = models.TextField(default='', blank=True)
-
-
 class Contribution(models.Model):
     """Model for contribution data."""
 
     user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='contributions')
     activity = models.ForeignKey(Activity, related_name='contributions')
-    location = models.ForeignKey(Location, related_name='contributions')
-    device = models.ForeignKey(Device, related_name='contributions')
+    location = models.ForeignKey(Location, related_name='contributions', null=True)
```

Comment:
This is for the same reason with the `contribution` foreign key. If we don't, then a data migration is on the way for a default value. Compared to the contribution field, a data migration for this case, is much easier though. Thoughts?